content/documentation/distributed-tracing/index.adoc

```diff
@@ -18,53 +18,73 @@ toc::[]
+We can interact with the jaeger UI with the external links, for example "View Trace in Tracing" goes to the Jaeger UI trace view.
 
-image::small_form.png[Smal Form Tracing View]
+image::trace_view_jaeger.png[Trace View]
 
+== External URL
 
-The service selector has the list of services of the selected namespaces organized by namespace.
+Configuring an external URL for Jaeger will enable more links from Kiali to Jaeger UI. This URL needs to be accessible from the browser (it's used for links generation). Example:
 
-image::service.png[Service Selector Tracing View]
+```yaml
+  external_services:
+    tracing:
+      in_cluster_url: 'http://tracing.istio-system/jaeger'
+      url: 'http://my-jaeger-host'
+```
 
+Once the url setting is set on the Kiali CR, Kiali will show...
 
-For an advanced search of traces we can expand the "Show advanced options."
+image::menu_external_link.png[Distributed Tracing View]
 
-image::full_form.png[Full form Tracing View]
+icon:bullhorn[size=2x]{nbsp} You may have `url` configured and not `in_cluster_url`, for instance, if Jaeger is not accessible from Kiali pod. In this situation, Kiali will not show its own traces chart but will display external links to the Jaeger UI instead.
 
+=== Other configuration
 
-* Tags: Empty by default, find the traces with these values. Example: `http.status_code=200`,error=true`
-* Limit results: 20 by default, number of traces results.
-* Span Configurations (More info about spans link:https://www.jaegertracing.io/docs/1.9/architecture/#span[here, window="_blank"])
-** Min Duration: Min duration for a span. Eg: 1.2s, 100ms, 500us
-** Max Duration: Max duration for a span. Eg: 1.2s, 100ms, 500us
+For advanced configuration on Jaeger integration, please refer to link:https://github.com/kiali/kiali/blob/c78f195fcbe214e56a918d616ed2f241be24c968/operator/deploy/kiali/kiali_cr.yaml#L433-L470[the Kiali CR description, window="_blank"]. It is relevant for config map as well, if you don't use the Kiali operator.
 
 
-== Kiali and Tracing Service have not the same protocol
+== Tracing Toolbar
 
-In this case you'll have a link to open the service in a new tab in your browser in the menu of Kiali and in the view of a specific service.
+Kiali has a toolbar to control this feature to facilitate the user experience with the tracing service. When we go to the tracing view we'll a form as the image below where we can select the traces interval, results limit, status code, errors, adjust time (expand results on time), last Xm traffic (Traces from last minutes) and refresh time.
+
+image::form.png[Form Tracing View]
+
+
+=== Tracing Results
+
+When user select a trace kiali shows the information related to that trace like number of spans, spans grouped by operation name, duration, date...
+
+image::kiali_trace.png[Trace view]
+
+In each span, tags and process are displayed.
```

Comment:
```suggestion
In each span, tags and processes are displayed.
```

Reply:
not sure about this one, is it 1 process per span or not?

Reply:
Yes , checked in Jaeger UI

Reply:
Yes checked in Jaeger UI